test/e2e/scheduling/preemption.go

```diff
@@ -307,8 +308,8 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 		}
 	})
 
-	ginkgo.It("runs ReplicaSets to verify preemption running path [Flaky]", func() {
-		podNamesSeen := make(map[string]struct{})
+	ginkgo.It("runs ReplicaSets to verify preemption running path", func() {
+		podNamesSeen := make(chan string, 6)
```

Comment:
is there a downside to making this way oversized (e.g. 100)? I don't want to block adds

Reply:
This's an additional gate to block the "under-preempted" case, but a little "brute forcing"... So no downside if we make it oversized.